src/api/options-misc.md

```diff
@@ -2,7 +2,7 @@
 
 ## name {#name}
 
-Explicitly declare a display name for the component.
+Explicitamente declara um nome de exibição para o componente.
 
 - **Type**
```

Comment:
"Type" needs to be translated

I recall seeing that missing on #54 as well

Reply:
@viniciusdeliz eu fundirei este pedido com o ramo principal,
e depois podemos corrigir e adicionar o que falta. :+1: